### PR TITLE
Add memorial service

### PR DESCRIPTION
**Pull request by tychonievich**
Tracks progress on #325

### CODE REVIEW

File: attribute-event-requests/INDI-event.md [hunk 1]
```diff
@@ -66,6 +67,32 @@ In [GEDCOM X](https://github.com/FamilySearch/gedcomx/blob/master/specifications
...
+
+- Obituaries
+- Memorial announcments and programmes
+
+### Absence
+
+The most closely related structures are:
+
+- `DEAT`: expresses that an individaul died, which generally has a different date than the memorial.
+- `BURI`: describes what was done with the remains of the individaul, which may or may not conincide with a memorial serive.
```
Comment (dthaler) [inline attribute-event-requests/INDI-event.md:88]:
```suggestion
- `BURI`: describes what was done with the remains of the individual, which may or may not coincide with a memorial serive.
```

**Review by dthaler — APPROVED** [review]
Ok other than typos

File: attribute-event-requests/INDI-event.md [hunk 2]
```diff
@@ -24,6 +24,7 @@ Guides for using this document can be found in the associated [README.md](README
 | `FCOM` | 4.0 | First Communion | |
 | `GRAD` | 4.0 | Graduation | |
 | `IMMI` | 3.0 | [Immigration](#immigration) | |
+| | | [Memorial Service](#mmorial-service) | |
```
Comment (tychonievich) [inline attribute-event-requests/INDI-event.md:27]:
```suggestion
| | | [Memorial Service](#memorial-service) | |
```

File: attribute-event-requests/INDI-event.md [hunk 3]
```diff
@@ -66,6 +67,32 @@ In [GEDCOM X](https://github.com/FamilySearch/gedcomx/blob/master/specifications
...
+### Description
+
+An event to mark or remember the death or presumed death of an individual, such as a funeral, wake, celebration of life, or memorial service.
+
+### Value
+
+Found in the following historical records:
+
+- Obituaries
+- Memorial announcments and programmes
```
Comment (tychonievich) [inline attribute-event-requests/INDI-event.md:81]:
```suggestion
- Memorial announcements and programs
- Church records
```

**Pull request merged**